token-service/src/main/java/org/p2p/token/service/database/TokenServiceDatabaseModule.kt

```diff
@@ -0,0 +1,24 @@
+package org.p2p.token.service.database
+
+import androidx.room.Room
+import org.koin.android.ext.koin.androidContext
+import org.koin.core.module.Module
+import org.koin.core.qualifier.named
+import org.koin.dsl.module
+import java.util.concurrent.Executors
+import org.p2p.core.common.di.InjectionModule
+
+internal const val TOKEN_SERVICE_DB_QUALIFIER = "TOKEN_SERVICE_DB_QUALIFIER"
+
+internal object TokenServiceDatabaseModule : InjectionModule {
+
+    override fun create(): Module = module {
+        single(named(TOKEN_SERVICE_DB_QUALIFIER)) {
```

Comment:
and why do you need named single here? is there multiple TokenServiceDatabase?

Reply:
just to be sure, that everything is OK )